services/challengedeployerservice/copyInPod.go

```diff
@@ -16,10 +16,6 @@ import (
 
 func CopyInPod(localFilePath string, pathInPod string) error {
 
-	if err := GetClient(g.KatanaConfig.KubeConfig); err != nil {
```

Comment:
Why?

Reply:
kubeclientset not being used here... function called unnecessarily